viewing_party/party.py

```diff
@@ -1,23 +1,263 @@
+#my first commit
+
 # ------------- WAVE 1 --------------------
 
+#creating the dictionary with (title, genre, rating) keys and returning the dict
 def create_movie(title, genre, rating):
-    pass
+    
+    dict_of_movies = {}
+    if title == None or genre  == None or rating == None:
+        return None
+    dict_of_movies["title"] = title
+    dict_of_movies["genre"] = genre
+    dict_of_movies["rating"] = rating
+    return dict_of_movies
+
+
+#the value of `user_data` is a dictionary with a key `"watched"`, 
+# and a value which is a list of dictionaries representing the movies the user has watched
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+#the value of `user_data` will be a dictionary with a key `"watchlist"`, 
+# and a value which is a list of dictionaries representing the movies the user wants to watch
+def  add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+#the value of `user_data` is a dictionary with a `"watchlist"` and a `"watched"`
+#If the title is in a movie in the user's watchlist:- remove that movie from the watchlist and 
+# add that movie to watched
+def watch_movie(janes_data, MOVIE_TITLE_1):
+    for movie in janes_data["watchlist"]:
+        if movie["title"]==MOVIE_TITLE_1:
+            watched_movie=movie
+            janes_data["watched"].append(watched_movie)
+            janes_data["watchlist"].remove(movie)
+            break

```

Comment:
Fantastic! Modifying a list as it's being iterated over is an "anti-pattern" that can cause a lot of bugs and errors. Breaking out of the list at this point ends the loop and eliminates the possibility for those bugs. Replacing the break statement with the return statement on line 41 would also work here.